src/panels/config/integrations/integration-panels/bluetooth/bluetooth-config-dashboard.ts

```diff
@@ -57,17 +108,66 @@ export class BluetoothConfigDashboard extends LitElement {
               >
             </div>
           </ha-card>
+          <ha-card
+            .header=${this.hass.localize(
+              "ui.panel.config.bluetooth.connection_slot_allocations_monitor"
+            )}
+          >
+            <div class="card-content">
+              ${this._renderConnectionAllocations()}
+            </div>
+          </ha-card>
         </div>
       </hass-subpage>
     `;
   }
 
+  private _getUsedAllocations = (used: number, total: number) =>
+    roundWithOneDecimal(getValueInPercentage(used, 0, total));
+
+  private _renderConnectionAllocations() {
+    if (this._connectionAllocationsError) {
+      return html`<div class="error">${this._connectionAllocationsError}</div>`;
```

Comment:
You may want to use an ha-alert instead 
``<ha-alert alert-type="error"> ${this._connectionAllocationsError}</ha-alert>```

Reply:
Thanks. Done